lua/wikis/counterstrike/Infobox/League/Custom.lua

```diff
@@ -380,7 +383,23 @@ end
 ---@return string?
 function CustomLeague:_createValveTierCell()
 	if self.valveTier then
-		return '[[' .. self.valveTier.link .. '|' .. self.valveTier.name .. ']]'
+		local showInfoIcon = self.data.endDate >= VALVE_TOR_START_DATE
+		return HtmlWidgets.Fragment{
```

Comment:
note for other reviewers: need fragment here so it is displayed in 1 line (concat with space) instead of using concat with `<br>` as infobox cell widget does

Reply:
oh right